### PR TITLE
Unity 2019.3 Upgrade

### PR DESCRIPTION
**Pull request by jamiebrynes7**
Upgraded the FPS to 2019.3. The vast majority of the changes are asset re-imports. 

Take note of https://github.com/spatialos/gdk-for-unity-fps-starter-project/commit/3b3fb3893ef658f3e5b93508cc11d72f81687020 which is required due to a behavioural difference in `OnValidate` and when things are serialized/deserialized. I wasted a day trying to track it down, but didn't feel its worth the time. 

Tech writers, you can find the changelog 👉 https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/263/commits/3ec550fc3246f868497681efdb22a991dec5f2fb

Tests:

- [x] Release QA pipeline
- [x] Windows client
- [x] MacOS client
- [x] iOS client

### CODE REVIEW

File: workers/unity/Assets/Fps/Scripts/Ragdoll/RagdollSpawner.cs [hunk 1]
```diff
@@ -48,23 +50,20 @@ private void OnDeath(HealthModifier deathDetails)
             ragdoll.LaunchRagdoll(deathDetails);
         }
 
-        private void OnValidate()
```
Comment (austinkelmore) [inline workers/unity/Assets/Fps/Scripts/Ragdoll/RagdollSpawner.cs]: Why does OnValidate not work anymore? Is it just for this object or Unity wide?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,18 +2,20 @@
...
 
-- Upgraded to GDK for Unity version `0.3.4`
 ## Breaking Changes
 
 - The FPS starter project now uses [query-based interest](https://docs.improbable.io/reference/14.4/shared/authority-and-interest/interest/query-based-interest-qbi) instead of [chunk-based interest](https://docs.improbable.io/reference/14.4/shared/authority-and-interest/interest/chunk-based-interest-cbi). [#253](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/253)
 - You must now pass in an `EntityId` to create a player `EntityTemplate`. [#254](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/254)
 
 ## Changed
 
+- Upgraded to GDK for Unity version `0.3.4`
```
Comment (austinkelmore) [inline CHANGELOG.md]: We're doing this for version 0.3.5, right?

Comment (jamiebrynes7) [reply CHANGELOG.md]: Yup, this was just me tidying up the previous release's changelog. There were 2 changed sections for some reason

Comment (paulbalaji) [reply CHANGELOG.md]: looking at the diff I think it's just moving this from above breaking changes into the `changed` section

**Comment by jamiebrynes7** [comment]
@anne-edwards added the full stops in https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/263/commits/f8e9fd6dec5354a334fa3a77dac3c10dc8151edf :)

**Pull request merged**